src/main/java/xyz/aadev/generitech/api/util/MachineTier.java

```diff
@@ -70,6 +77,16 @@ public static MachineTier byMeta(int meta) {
         return MachineTier.values();
     }
 
+    public static MachineTier[] allexeptTier_0() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS00100)